README.md

```diff
@@ -127,6 +128,13 @@ Examples:
   * To pass your email and passowrd:  
         * `peloton-to-garmin.py -email you@email.com -password mypassword`  
 
+### Ensure activities are counting towards Garmin challenges and badges
+To be able to have all synced activities counting towards Garmin challenges and badges, data from your Garmin hardware is needed. For getting it, please track one workout directly with your hardware (like a Garmin watch) and sync it with Garmin Connect.
+
+After done this, please export this activity as a TCX file and copy out ```<Author>...</Author>``` and ```<Creator>...</Creator>``` into ```author_info.xml``` and ```device_info.xml```.
+
+You can find samples of hwo these files are looking like at ```author_info.sample.xml``` and ```device_info.sample.xml```.
+
```

Comment:
This is one thing I wanted to understand better and research more before pulling in this change.  **Is it actually necessary for this to match the users device or could would it work with any valid device we configure?**

If this feature still works when given any device then my thought is that P2G should automatically include a working default for Author and DeviceInfo that matches a real device. Then we should provide a means for users to override that to match their actual device if they want to.

Reply:
Great questions.  I do not know.  I only tested my device information

Reply:
I also don't know - but it seems these are actually real device manufactoring numbers - I would not encourage using a default one for compliance / warranty reasons

Reply:
I would caution, that if someone is concerned about compliance/warranty then they should not use P2G. Neither Garmin nor Peloton publish a public api (peloton has already made me reset my password once or twice for _suspicous activity_) .  Everything that is done by this program is through reverse engineering and working around the system.

The other reason it would be good to understand more about how the Author and Creator sections work is that there is an existing request #38 to support power zones on the workout. Currently, Garmin Connect only allows you to set personal power zone information if you own a device capable of tracking power.  Many people just have activity trackers from Garmin, so they are not able to set their power zone info today.   But if we could provide them with a working default device config in P2G then it may pave the way for P2G to also track power zones.

Reply:
My guess on the power zone question would be to find a TCX file from Garmin where a power device was used and then you can simulate what they are doing in it.  My guess is the creator may have an additional element with a power meter device.  Not sure though as I do not have one.